src/main/webapp/i18n/de/cleanupService.json

```
@@ -15,13 +15,49 @@
             "deleteOldSubmissionVersions": "Alte Teilnahmeversionen löschen",
             "deleteOldFeedback": "Altes Feedback löschen"
         },
+        "tooltip": {
+            "deleteOrphans": "Löscht verwaiste Entitäten, die nicht mehr mit gültigen Ergebnissen oder Teilnahmen verbunden sind. Dazu gehören Feedback, Textblöcke und Bewertungen, die sich auf ungültige Ergebnisse, Teilnahmen oder Einreichungen beziehen.",
+            "deletePlagiarismComparisons": "Löscht Plagiatsvergleiche mit dem Status 'Keine', die zu Kursen innerhalb des angegebenen Datumsbereichs gehören.",
```

I do not have a super strong opinion here, it might make sense not to translate this value 
```suggestion
            "deletePlagiarismComparisons": "Löscht Plagiatsvergleiche mit dem Status 'None', die zu Kursen innerhalb des angegebenen Datumsbereichs gehören.",
```

---

I agree on this one tbh. `Keine` sounds weird here